doc/source/how-to/dns-configuration.rst

```diff
@@ -0,0 +1,74 @@
+DNS configuration
+=================
+
+As explained in :ref:`Documenting`, PyAnsys projects publish their documentation
+online under the following canonical name (CNAME) convention:
+
+``https://<product>.docs.pyansys.com``
+
+In order to request a CNAME for the ``pyansys.com`` domain, please contact
+`Maxime Rey`_, `Roberto Pastor Muela`_ or `Alex Kaszynski`_. Any of these members
+can handle the creation of the requested PyAnsys subdomain.
+
+Once the CNAME is created, repository administrators can configure their published
+documentation in GitHub pages to be exposed through it. In order to configure the CNAME
```

Comment:
Maybe here we could specify that in one hand the CNAME is located the `gh-pages` branch.
But on the other hand the Custom domain is also mentioned in GitHub repository `Settings/Pages `.

Talking about those two distinct locations where the custom domain is mentioned might help to clarify the process because I noticed some confusion for internal users.

Maybe not the perfect location to talk about it but it might be helpful.

Reply:
CNAME configuration is done through Settings/Pages --> Custom domain.

Once you do that, GitHub automagically creates the ``CNAME`` file in your ``gh-pages`` branch without you having to do anything... Or in whatever branch your docs are published. Remember that the naming convention ``gh-pages`` is only that, a convention. You could potentially publish your docs under a branch called ``my-docs-branch``, for example.

I don't know if it's worth mentioning that the CNAME is also located in the ``gh-pages`` branch or not honestly.. because admins won't have to do anything there. It's done for them when you configure it in the Settings/Pages site. And if you change it, it also gets updated in the ``gh-pages`` branch.

But I'm open to discussion honestly 😄

Reply:
You are correct, thinking twice about it, it could bring more confusion.

Reply:
Also, you can publish now with a CI action without having a branch at all.

Reply:
Yep, that's also true. GH evolves fast 😄